yt/data_objects/construction_data_containers.py

```diff
@@ -256,7 +263,7 @@ def get_data(self, fields=None):
         projected_units = self.comm.mpi_bcast(self._projected_units)
         self._projected_units = projected_units
         # Note that this will briefly double RAM usage
-        if self.method in ["mip", "max"]:  # TODO: suggest mip as deprecated
+        if self.method == "max":
```

Comment:
it's important to maintain backward compatibility as long as the deprecated feature isn't actually removed, so we have to keep checking for `"mip"` here for now

Reply:
Okay yeah I wasn't sure if that'd be confusing so I put "mip" back in everywhere :)

Reply:
maintaining deprecated code is confusing indeed, but it's necessary to avoid breaking everyone all the time :)

Reply:
Wellll, note that I set method to "max" if method is "mip" (similar to how the value of style gets set to method) so it shouldn't break removing "mip" elsewhere (theoretically...)

Reply:
yeah I saw that, thank you ! I was just making a case.